Server/src/Server.Application/CommandHandlers/AddAuthorHandler.cs

```diff
@@ -0,0 +1,40 @@
+using FluentValidation;
+using MediatR;
+using Server.Application.InfrastructureInterfaces;
+using Server.Domain.Entities;
+using Server.Domain.Repositories;
+
+namespace Server.Application.CommandHandlers;
+
+//TODO Add validators for other commands and queries 
+public sealed class AddBookHandlerValidator : AbstractValidator<AddAuthorCommand>
+{
+    public AddBookHandlerValidator()
+    {
+        RuleFor(x => x.FirstName).NotEmpty().MaximumLength(40);
+        RuleFor(x => x.LastName).NotEmpty().MaximumLength(50);
+    }
+}
+
+public sealed record AddAuthorCommand(Guid Id, string FirstName, string LastName) : IRequest;
```

Comment:
imo commandy nie powinny przyjmować Guidów, powinny one być tworzone na backendzie lub przez samą bazę danych jak SQL Server to wgl wspiera